### ISSUE TITLE
Is there any way to edit these curves in browser.

### ISSUE THREAD

**Issue by guptaashwanee** [elfalem/Leaflet.curve#51]

**Comment by elfalem** [comment]
Yes, it is possible to use Leaflet.draw plugin to edit the curves. However, do note that the plugin seems to be abandoned and the request to merge the changes for editing curves was not merged. It can be used from https://github.com/qpincon/Leaflet.draw/tree/curve_built. See #32 for more details.
